src/sentry/profiles/task.py

```diff
@@ -137,6 +138,7 @@ def encode_payload(message: dict[str, Any]) -> str:
             times=2,
             delay=5,
         ),
+        compression_type=CompressionType.ZSTD,
```

Comment:
I think we'll end up double zipping here as the profiles consumer will do one zip, and then the taskworker system will zip again. This isn't ideal, but as an intermediary step it should be ok. We should test this locally to make sure we're not going to break the tasks.